packages/cyberstorm/src/components/MetaItem.tsx

```diff
@@ -4,23 +4,23 @@ import styles from "./componentStyles/MetaItem.module.css";
 export interface MetaItemProps {
   label?: string;
   icon?: ReactNode;
-  metaItemStyle?: string;
+  metaItemStyle?: "metaItem__default" | "metaItem__green";
 }
 
 /**
  * Cyberstorm MetaItem component
  */
 export const MetaItem: React.FC<MetaItemProps> = (props) => {
   const { label, icon, metaItemStyle } = props;
-  styles; //if styles is not called, the classes from the css module aren't found
-  const additionalStyle = metaItemStyle
-    ? " metaItem__" + metaItemStyle
-    : " metaItem__default";
 
   return (
-    <div className={"metaItem" + additionalStyle}>
+    <div
+      /* TS is not aware of defaultProps of function components. */
```

Comment:
There's no `defaultProps` defined for this component? And since `metaItemStyle` is marked optional parameter, we really shouldn't use `!` for non-null assertion, since the value can indeed be nullish without default prop definition. The proper fix would be to add `metaItem__default` as the default prop though.

Reply:
added a defaultProp for the style